packages/hsl-shared-components/src/NewsFeed/NewsFeedItem.js

```diff
@@ -54,28 +58,37 @@ const NewsFeedItem = ({
   title,
   image,
   className
-}) => (
-  <StyledLi className={className}>
-    <TextContainer className="text-container">
-      <Span className="top">
-        <Span className="category">{ category }</Span>
-        <Span> — </Span>
-        <Span>{ moment(timestamp).calendar() }</Span>
-      </Span>
-      <Span className="bottom">
-        { title }
-      </Span>
-    </TextContainer>
-    { image && <Image src={image} alt="news image" /> }
-  </StyledLi>
-)
-;
+}) => {
+  const imageElement = React.isValidElement(image) ? image : <Image src={image} alt={title} />;
+  return (
+    <StyledLi className={className}>
```

Comment:
Not really part of this task, but className could be changed to ...rest to avoid some hard to find bugs.

Reply:
I think we should get rid of those classnames for good
i don't think it does anything anymore either ,as i changed all them to use styled components